h2/src/main/org/h2/util/JdbcUtils.java

```diff
@@ -294,7 +290,12 @@ public static Connection getConnection(String driver, String url,
         } else {
             Class<?> d = loadUserClass(driver);
             if (java.sql.Driver.class.isAssignableFrom(d)) {
-                return DriverManager.getConnection(url, prop);
+                try {
```

Comment:
I'd like a comment here - when reading code, it's always nice to see a reason for anything weird.
thanks.

Reply:
@grandinj thanks for review